src/diffusers/models/attention.py

```diff
@@ -258,6 +260,8 @@ class AttentionBlock(nn.Module):
         norm_num_groups (`int`, *optional*, defaults to 32): The number of groups to use for group norm.
         rescale_output_factor (`float`, *optional*, defaults to 1.0): The factor to rescale the output by.
         eps (`float`, *optional*, defaults to 1e-5): The epsilon value to use for group norm.
+        cross_attention_dim (`int`, *optional*):
```

Comment:
Actually we cannot really do this because by doing so we start having two competing cross attention classes, ideally we should only have one attention class. Let's try to put this instead into the `CrossAttention` class.

Reply:
I'll help you to merge this into the CrossAttention class => we should quite soon get rid of this class all together.